Realm/Realm.Sync/Handles/SharedRealmHandleExtensions.cs

```diff
@@ -45,7 +45,7 @@ private static class NativeMethods
 
             public unsafe delegate void RefreshAccessTokenCallbackDelegate(IntPtr session_handle_ptr);
 
-            public unsafe delegate void SessionErrorCallback(IntPtr session_handle_ptr, ErrorCode error_code, sbyte* message_buf, IntPtr message_len, IntPtr user_info_pairs, int user_info_pairs_len);
+            public unsafe delegate void SessionErrorCallback(IntPtr session_handle_ptr, ErrorCode error_code, byte* message_buf, IntPtr message_len, IntPtr user_info_pairs, int user_info_pairs_len);
```

Comment:
I can't say I'm 100% sure :) The thing is, there's no longer a string ctor that accepts `sbyte*` in .NET standard, but converting to `byte*` and using `Encoding.GetString` seems to do the trick (passes the tests at least).